extensions/amp-story-panning-media/0.1/amp-story-panning-media.js

```diff
@@ -182,8 +190,7 @@ export class AmpStoryPanningMedia extends AMP.BaseElement {
    * @param {!Object<string, string>} panningMediaState
    */
   onPanningMediaStateChange_(panningMediaState) {
-    if (panningMediaState && this.distance_ <= 1) {
-      // TODO(#31932): Update siblings that are part of the same group.
+    if (panningMediaState[this.groupId_] && this.distance_ <= 1) {
```

Comment:
sorry I must have missed something but why do we want to animate for neighbor pages?

Reply:
This ensures the frame on the next page is in the correct position. 
When only updating the active page, a noticeable flash of an incorrectly frame is visible:
![Jan-28-2021 12-36-45](https://user-images.githubusercontent.com/3860311/106176652-9687b780-6165-11eb-9fa6-da94b50fb7c7.gif)

Reply:
Gotcha, thanks for the clarification and gif! super helpful